pkg/config/capabilities.go

```diff
@@ -57,7 +57,150 @@ func (c *ConfigTx) GetApplicationCapabilities() (map[string]bool, error) {
 	return capabilities, nil
 }
 
-func getCapabilities(configGroup *cb.ConfigGroup) (map[string]bool, error) {
+// AddChannelCapability adds capability to the provided channel config.
```

Comment:
All of these godoc comments seem out of date at this point. We need a follow up to address these (if there isn't one already).